src/rules/no-default-export.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  meta: {
+    docs: {},
+  },
+
+  create: function (context) {
+    // ignore non-modules
+    if (context.parserOptions.sourceType !== 'module') {
+      return {}
+    }
+
+    return {
+      ExportDefaultDeclaration: function (ast) {
+        context.report({
+          node: ast,
+          message: 'Prefer named exports.',
+        })
+      },
+
+      ExportNamedDeclaration: function (ast) {
+        for (var i = 0; i < ast.specifiers.length; i++) {
+          var node = ast.specifiers[i]
+
+          if (node.type === 'ExportDefaultSpecifier') {
+            context.report({
+              node: ast,
+              message: 'Prefer named exports.',
+            })
+          } else if (node.type === 'ExportSpecifier' &&
+              node.exported.name === 'default') {
+            context.report({
+              node: ast,
+              message: 'Prefer named exports.',
```

Comment:
Can you extract this message to an external variable? Naming it `message` would it be destructured in.